mongo/src/main/java/com/redhat/lightblue/mongo/metadata/BSONParser.java

```diff
@@ -51,140 +51,86 @@
 import com.redhat.lightblue.util.JsonUtils;
 
 @SuppressWarnings({"rawtypes", "unchecked"})
-public class BSONParser extends MetadataParser<BSONObject> {
+public class BSONParser extends MetadataParser<Object> {
     private static final Logger LOGGER = LoggerFactory.getLogger(BSONParser.class);
 
     public static final String DELIMITER_ID = "|";
 
-    public BSONParser(Extensions<BSONObject> ex,
+    public BSONParser(Extensions ex,
```

Comment:
Fixed.

On Fri, Apr 1, 2016 at 8:41 AM, Dennis Crissman <notifications@github.com>
wrote:

> In mongo/src/main/java/com/redhat/lightblue/mongo/metadata/BSONParser.java
> <https://github.com/lightblue-platform/lightblue-mongo/pull/232#discussion_r58215156>
> :
>
> >      private static final Logger LOGGER = LoggerFactory.getLogger(BSONParser.class);
> >
> >      public static final String DELIMITER_ID = "|";
> >
> > -    public BSONParser(Extensions<BSONObject> ex,
> > +    public BSONParser(Extensions ex,
>
> Should include generic type of Object
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-mongo/pull/232/files/b6bba7e856c5d1d000a696ddd1d634f844bf77c1#r58215156>
>